tests/integration/test_mongodb.py

```diff
@@ -55,16 +56,10 @@ async def test_deploy(
 @only_with_juju_secrets
 async def test_deploy_and_relate_mongodb(ops_test: OpsTest, cloud_name: str):
     """Test the relation with MongoDB and database accessibility."""
-    if (await ops_test.model.get_status()).model.version.startswith(
-        "3.4."
-    ) and cloud_name == "microk8s":
-        pytest.skip("Test is incompatible with Juju 3.4 on microk8s")
-
-    channel = "5/edge" if cloud_name == "localhost" else "edge"
     await asyncio.gather(
         ops_test.model.deploy(
             MONGODB[cloud_name],
-            channel=channel,
+            channel="6/edge",
```

Comment:
Both k8s and VM have a 6/edge branch now

Reply:
Uff, we were testing here deprecated 5/edge. cc: @Mehdi-Bendriss

Reply:
I think juju 3.1 was installing the correct revision https://github.com/canonical/data-integrator/actions/runs/10875992483/job/30175392118#step:31:114

So we didn't see that the channel was wrong.